components/Typography/examples/TypographyMaterialStylesViewController.m

```diff
@@ -44,21 +45,46 @@ - (void)viewDidLoad {
   _styleNames = @[
                   // Common UI fonts.
                   @"Headline Font",
+                  @"Headline Font (Dynamic Type-enabled)",
                   @"Title Font",
+                  @"Title Font (Dynamic Type-enabled)",
                   @"Subhead Font",
+                  @"Subhead Font (Dynamic Type-enabled)",
                   @"Body 2 Font",
+                  @"Body 2 Font (Dynamic Type-enabled)",
                   @"Body 1 Font",
+                  @"Body 1 Font (Dynamic Type-enabled)",
                   @"Caption Font",
+                  @"Caption Font (Dynamic Type-enabled)",
                   @"Button Font",
+                  @"Button Font (Dynamic Type-enabled)",
 
                   // Display fonts (extra large fonts)
                   @"Display 1 Font",
+                  @"Display 1 Font (Dynamic Type-enabled)",
                   @"Display 2 Font",
+                  @"Display 2 Font (Dynamic Type-enabled)",
                   @"Display 3 Font",
-                  @"Display 4 Font"
+                  @"Display 3 Font (Dynamic Type-enabled)",
+                  @"Display 4 Font",
+                  @"Display 4 Font (Dynamic Type-enabled)"
                   ];
 
-  _styleFonts = @[
+  _styleStandardFonts = @[
```

Comment:
why create two separate arrays? just put them interlaced like the names?

Reply:
Simplification!
Done.